spec/models/project_spec.rb

```diff
@@ -83,7 +83,8 @@
   end
 
   describe '#related_projects' do
-    let(:organization) { Organization.create!(name: 'CodeMontage') }
+    let(:organization) { Organization.create!(name: 'CodeMontage', 
+                                      github_org: 'codeMontage') }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 87, 66 should be on its own line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.